module/VuFind/src/VuFind/Search/SearchTabsHelper.php

```diff
@@ -72,6 +72,13 @@ class SearchTabsHelper extends \Laminas\View\Helper\AbstractHelper
      */
     protected $permissionConfig;
 
+    /**
+     * Tab detailed configuration
+     *
+     * @var array
+     */
+    protected $detailConfig;
```

Comment:
If we decide to change the name of the .ini setting, we might also want to review the naming of this property and related code. Maybe a simple `getOptions` or `getSettings` would be more concise than `getTabDetailConfig`. At very least, I don't think we need Tab in the method name since the method belongs to a helper with "Tabs" in the name.

Reply:
I opted for getSettings.